kuma/search/views.py

```diff
@@ -133,12 +132,13 @@ def get_filters(self, aggregations):
 # (see http://breachattack.com/). It still needs to allow a user to click
 # the filter switches (bug 1426968).
 # Alternate: forbid gzip by setting Content-Encoding: identity
-search = ratelimit(key='user_or_ip', rate='25/m', block=True)(
+search = never_cache(ratelimit(key='user_or_ip', rate='25/m', block=True)(
     SearchView.as_view()
-)
+))
 
 
-@cache_page(60 * 15)  # 15 minutes.
+@never_cache
+@cache_page(60 * 15)  # Cache server-side (memcached) for 15 minutes.
```

Comment:
I think ``@never_cache`` is wrong here. ``@cache_page`` should be replaced with a more standard cache header.

Also ``suggestions`` is a dumb view and I feel dumber for having read it.